pkg/skaffold/deploy/helm/helm.go

```diff
@@ -68,25 +68,13 @@ import (
 )
 
 var (
-	// versionRegex extracts version from "helm version --client", for instance: "2.14.0-rc.2"
```

Comment:
nit: perhaps we can add a comment stating that the helm version is no longer relevant/needs-checking in V2 (vs V1) and give a brief explanation